src/sidebar/components/test/tag-editor-test.js

```diff
@@ -339,4 +340,12 @@ describe('TagEditor', function() {
       assert.isTrue(fakeTagsService.filter.calledWith(''));
     });
   });
+
+  // FIXME-A11Y
```

Comment:
This refers to the `it.skip` on the next line. If you run the full test suite you'll see "3 tests skipped" at the end of the output.

Unfortunately Mocha doesn't have a way to differentiate an expected failure that I know of (which is what I'd like to use) rather than a test that is merely "skipped" for arbitrary reasons.

Reply:
understood